examples/com/venafi/vcert/sdk/example/policymanagement/policy_specification.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "",
+  "policy": {
+    "domains": [
+      "venafi.com",
+      "kwan.com"
+    ],
+    "maxValidDays": 120,
+    "wildcardAllowed": false,
+    "certificateAuthority": "ENTRUST\\Entrust\\STANDARD_SSL",
```

Comment:
Do this CA exists? If the file is expected to be used, then set a valid CA.

Reply:
Yes, it exists in Cloud